style/typography.js

```diff
@@ -1,38 +1,43 @@
 import styled from 'styled-components';
+import { min } from '../lib/responsive';
 const fonts = {
   swearDisplay: 'swear-display, serif',
   poppins: 'poppins, sans-serif',
 };
 
 const fontSizes = {
-  h1: 'clamp(2.5em, 8vw, 8em)',
-  h2: 'clamp(1.8em, 5vw, 6em)',
-  h3: 'clamp(1em, 2.5vw, 3em)',
-  p: 'clamp(.6em, 1.5vw, 2em)',
+  h1: 'clamp(11em, 25vw, 14em)',
+  h2: 'clamp(5em, 10vw, 8em)',
+  h3: 'clamp(3em, 5vw, 4em)',
 };
 
 const fontStyles = {
   italic: 'italic',
 };
 
 const fontWeights = {
-  normal: `400px`,
-  medium: `700px`,
-  bold: `900px`,
+  normal: `400`,
+  medium: `600`,
+  bold: `650`,
 };
 
-const H1 = styled.h1`
+const H1Italic = styled.h1`
```

Comment:
Don't think we need a separate H1 Italic
![Screen Shot 2021-04-05 at 11 41 25 AM](https://user-images.githubusercontent.com/33647880/113593047-f25a3b00-9603-11eb-81ad-39e65aa79022.png)
![Screen Shot 2021-04-05 at 11 41 30 AM](https://user-images.githubusercontent.com/33647880/113593052-f4bc9500-9603-11eb-9f46-c18bb968a87c.png)